site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -236,7 +236,14 @@ The following tables lists the configurable parameters of the Agones chart and t
 
 | Parameter                                           | Description                                                                                     | Default                |
 | --------------------------------------------------- | ----------------------------------------------------------------------------------------------- | ---------------------- |
-|                                                     |                                                                                                 |                        |             
+| `agones.controller.annotations`                     | Controller [annotations][annotations] settings for pod assignment                               | `{}`                   |
```

Comment:
```suggestion
| `agones.controller.annotations`                     | [Annotations][annotations] added to the Agones controller pods                               | `{}`                   |
```

Just going by Robbie's suggestion, to make it super clear. WDYT?

Reply:
Fixed.